### PR TITLE
Disable all PEVerification until #6190 is resolved

### PR DESCRIPTION
**Pull request by davkean**
Disabling PEVerification until #6190 is resolved

### CODE REVIEW

**Comment by davkean** [comment]
tag @dotnet/roslyn-compiler @dotnet/roslyn-infrastructure.

**Comment by davkean** [comment]
tag @tmat

File: src/Test/Utilities/Desktop/HostedRuntimeEnvironment.cs [hunk 1]
```diff
@@ -799,7 +799,9 @@ public string[] PeVerifyModules(string[] modulesToVerify, bool throwOnError = tr
             // for ARM testing at present.
             StringBuilder errors = new StringBuilder();
             List<string> allOutput = new List<string>();
-#if !(ARM)
+
+// Disable all PEVerification due to https://github.com/dotnet/roslyn/issues/6190
+#if false
```
Comment (amcasey) [inline src/Test/Utilities/Desktop/HostedRuntimeEnvironment.cs]: Consider leaving the old text as a comment to make this easier to re-enable.

**Comment by amcasey** [comment]
:+1:

**Pull request merged**